docs/v1/P4Runtime-Spec.adoc

```diff
@@ -1939,18 +1759,19 @@ According to the P4 specification, the type parameter of a Value Set, which
 defines the type of the expression that can be matched against the Value Set in
 a parser transition, and therefore determines the format of the members that can
 be inserted into the Value Set by the control plane, must be one of `bit<W>`,
-`tuple`, or `struct` [@P4SelectExpr]. The rest of this section looks at all 3 of
+`tuple`, or `struct` cite:[P4SelectExpr]. The rest of this section looks at all 3 of
 these cases and gives an example `ValueSet` message when appropriate.
 
-1. If the type parameter is `bit<W>`, `match` will include exactly one
+. If the type parameter is `bit<W>`, `match` will include exactly one
    `MatchField` message, with the following fields (if a field is omitted here,
    it means the default Protobuf value should be used):
 
  * `id`: set to 1
  * `bitwidth`: set to the value of `W`
  * `match_type`: set to `EXACT`
 
-~ Begin P4Example
+[source,p4]
+----
```

Comment:
The lines `~ End P4Example` and `~ Begin Prototext` below need to be updated to Asciidoc markup